### PR TITLE
Customizations: Add suppressUpdates

### LINKED ISSUE #10513: Customizations: perf requests
For the Customizations framework, I'd like to see two things:

1) A way to batch updates. For example, you can only apply one scoped setting at a time. If you do this in a for loop, then ever single iteration is triggering a forced update. On a SharePoint page this freezes the whole page for almost 1s on Chrome. So either a batchedUpdate() utility function, or a way to set multiple settings and scopedSettings at one time (similar to how you can do it in <Customizer>).
2) A way to set Customizations _without_ triggering an update. Since the forceUpdate can be very expensive, I might want to defer it to a later time to trigger manually, or not trigger one at all because I know the Customization won't affect anything that's currently rendered (but newly rendered components will pick it up).

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #10513
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Adds suppressUpdates() to Customizations, allowing you to set many settings, such as in a for loop, without triggering updates.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10549)

### CODE REVIEW

File: packages/utilities/src/customizations/Customizations.ts [hunk 1]
```diff
@@ -32,18 +32,24 @@ let _allSettings = GlobalSettings.getValue<ICustomizations>(CustomizationsGlobal
...
+  private static _suppressUpdates: boolean;
+
   public static reset(): void {
     _allSettings.settings = {};
     _allSettings.scopedSettings = {};
   }
 
-  // tslint:disable-next-line:no-any
+  /** Apply global Customization settings.
+   * @example Customizations.applySettings(\{ theme: \{...\} \});
```
Comment (phkuo) [inline packages/utilities/src/customizations/Customizations.ts]: TSLint made me escape all these things... not sure if there's a better way to do it, the intellisense comments in VSCode don't look right

**Comment by phkuo** [comment]
@msft-github-bot automerge pending approval from David Zearing

**Comment by phkuo** [comment]
@msft-github-bot automerge once David Zearing has approved

**Comment by phkuo** [comment]
@msft-github-bot automerge when @dzearing has signed off

File: packages/utilities/src/customizations/Customizations.ts [hunk 2]
```diff
@@ -71,6 +76,18 @@ export class Customizations {
...
 
+  /** Used to run some code that sets Customizations without triggering an update.
+   * Useful for applying Customizations that don't affect anything currently rendered.
+   */
+  public static suppressUpdates(code: () => void): void {
+    Customizations._suppressUpdates = true;
+    try {
+      code();
+    } finally {
+      Customizations._suppressUpdates = false;
```
Comment (dzearing) [inline packages/utilities/src/customizations/Customizations.ts]: Hmm. it will suppress updates, but then, when will the updates raise changes?

Comment (dzearing) [reply packages/utilities/src/customizations/Customizations.ts]: (I expected that we'd track something like a pending change required, and then at the end if that's true, raise change.

Comment (dzearing) [reply packages/utilities/src/customizations/Customizations.ts]: Suggested tweak:

```tsx
public static applyBatchedUpdates(cb: () => void, supressChanges: false)
```

**Pull request merged**